docs/strategy-callbacks.md

```diff
@@ -760,21 +760,30 @@ The `position_adjustment_enable` strategy property enables the usage of `adjust_
 
-`max_entry_position_adjustment` property is used to limit the number of additional entries per trade (on top of the first entry order) that the bot can execute. By default, the value is -1 which means the bot have no limit on number of adjustment entries.
-
-The strategy is expected to return a stake_amount (in stake currency) between `min_stake` and `max_stake` if and when an additional entry order should be made (position is increased -> buy order for long trades, sell order for short trades).
-If there are not enough funds in the wallet (the return value is above `max_stake`) then the signal will be ignored.
 Additional orders also result in additional fees and those orders don't count towards `max_open_trades`.
 
 This callback is **not** called when there is an open order (either buy or sell) waiting for execution.
 
 `adjust_trade_position()` is called very frequently for the duration of a trade, so you must keep your implementation as performant as possible.
 
+Position adjustments will always be applied in the direction of the trade, so a positive value will always increase your position (negative values will decrease your position), no matter if it's a long or short trade.
+
+Modifications to leverage are not possible, and the stake-amount returned is assumed to be before applxying leverage.
+
+### Increase position
+
+The strategy is expected to return a positive stake_amount (in stake currency) between `min_stake` and `max_stake` if and when an additional entry order should be made (position is increased -> buy order for long trades, sell order for short trades).
+
+If there are not enough funds in the wallet (the return value is above `max_stake`) then the signal will be ignored. 
+`max_entry_position_adjustment` property is used to limit the number of additional entries per trade (on top of the first entry order) that the bot can execute. By default, the value is -1 which means the bot have no limit on number of adjustment entries.
+
 Additional entries are ignored once you have reached the maximum amount of extra entries that you have set on `max_entry_position_adjustment`, but the callback is called anyway looking for partial exits.
 
-Position adjustments will always be applied in the direction of the trade, so a positive value will always increase your position (negative values will decrease your position), no matter if it's a long or short trade.
+### Decrease position
 
-Modifications to leverage are not possible, and the stake-amount returned is assumed to be before applying leverage.
+The strategy is expected to return a negative stake_amount (in stake currency) for a partial exit.
+Returning `-trade.stake_amount` results in a full exit.
+Returning a value more than `-trade.stake_amount` (so remaining stake_amount would become negative) will result in the bot ignoring the signal.
```

Comment:
@xmatthias won't this line incorrect since in current code, the bot will reduce the amount to match trade_amount, which means the exit will be successful in the end

Reply:
partially - the idea was to make this a hard requirement.

i removed that part in the backtesting code - but apparently forgot to remove it in the live section ...